### ISSUE TITLE
decoder_outputs_train have different dim!

### ISSUE THREAD

**Issue by yanwii** [ematvey/tensorflow-seq2seq-tutorials#6]
```
            (self.decoder_outputs_train,
             self.decoder_state_train,
             self.decoder_context_state_train) = (
                seq2seq.dynamic_rnn_decoder(
                    cell=self.decoder_cell,
                    decoder_fn=decoder_fn_train,
                    inputs=self.decoder_train_inputs_embedded,
                    sequence_length=self.decoder_train_length,
                    time_major=self.time_major,
                    scope=scope,
                )
            )
```

i got some different dim of decoder_outputs_train.

Traceback liks this:
```
InvalidArgumentError (see above for traceback): logits and labels must have the same first dimension, got logits shape [500,1546] and labels shape [550]
```
if the format of my train data is wrong ? 
the labels' shape is 550!
